packages/framework-integration-tests/integration/providers/aws/auth.integration.ts

```diff
@@ -33,7 +33,7 @@ describe('With the auth API', () => {
     mockCartId = random.uuid()
   })
 
-  context('an internet rando', () => {
+  context('an internet random', () => {
```

Comment:
Actually the correct term is rando (Source: https://www.merriam-webster.com/dictionary/rando) 😅

Reply:
👍 
```suggestion
  context('an internet rando', () => {
```